bridges/snowbridge/parachain/runtime/kusama-common/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
```

Comment:
This will probably fail the CI license header check.

Reply:
Are you referring to this CI license header check? https://github.com/paritytech/polkadot-sdk/actions/runs/7466538622/job/20318108907?pr=2894

Reply:
Yea, but looks like its green. Dont know, maybe we can still use the same Apache 2.0 header than in other files (minus the first line that mentions parity).